extensions/twitter/README.md

```diff
@@ -0,0 +1,28 @@
+# Twitter extension
+This document defines how to use `twitter` extension in AsyncAPI documents.
+
+## Overview 
+This extension allows you to place the twitter account in charge of the team/company of the API.
+
+## Extension Definition
+Extensions are the mechanism AsyncAPI has to allow you use custom or protocol-specific features. Extensions follow a separate release cycle and everyone can create their own. This repository is meant to contain a list of official and community supported extensions.
```

Comment:
it is not twitter related info, please remove

it is ok if this is empty, idea is that it is `h2` as it just introduces the following `h3`

Reply:
So we don't need to have extension definition here?

Reply:
you mean what extensions are in genera? no

this header says that below you have extension definition, which is its type and supported location

Reply:
ok got it , AI added the extension location to h3